Dockerfile

```diff
@@ -1,44 +1,94 @@
-FROM python:3.13-alpine3.21
+FROM node:20.19-alpine AS node-deps
```

Comment:
what version(s) of `node` should this be built with?

Reply:
i dont really care, I usually use some LTS version for years and once it runs out upgrade. I think I have tried everything with 22 on some machine so that should work